tests/installation/user_settings.pm

```diff
@@ -34,13 +34,19 @@ sub run {
         return;
     }
 
-    $self->enter_userinfo();
-    assert_screen 'inst-userinfostyped';
-    if (match_has_tag 'boo#1122804') {
-        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
-        $self->enter_userinfo(retry => 1);
-        assert_screen 'inst-userinfostyped';
-    }
+    # retry if not typed correctly
+    my $max_tries = 2;
+    my $i = 0;
+    do {
+        $self->enter_userinfo();
+        assert_screen qw(inst-userinfostyped-ignore-full-name inst-userinfostyped-expected-typefaces);
+        if (!match_has_tag('inst-userinfostyped-expected-typefaces')) {
+            record_info("poo#46190', 'Typing issue with fullname - poo#46190 (Tried $i times)");
+            $self->enter_userinfo(retry => 1);
```

Comment:
I think this second call to enter_userinfo is a bit out of place, specially because we're inside a loop

Reply:
No, I just forgot to remove it